.travis-ipython4-requirements.txt

```diff
@@ -0,0 +1,22 @@
+Jinja2>=2.7.3
```

Comment:
This probably doesn't need a whole separate file.  If we just specify `IPython>=3.2.1` in the requirements, the travis build can just do

`pip install IPython==$IPYTHON_VERSION && pip install -r requirements.txt`